rules/DowngradePhp80/Rector/FuncCall/DowngradeStrContainsRector.php

```diff
@@ -77,17 +70,14 @@ public function refactor(Node $node): Identical | NotIdentical | null
             return null;
         }
 
-        if (! $this->argsAnalyzer->isArgsInstanceInArgsPositions($funcCall->args, [0, 1])) {
```

Comment:
`if ($funcCall->isFirstClassCallable()) {` should be checked early

Reply:
Same count check here too